Art/maulikatgit/style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  margin: 0;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-color: #000;
+}
+
+.diamond {
+  width: 100px;
+  height: 100px;
+  position: relative;
+  transform: rotate(90deg);
+}
+
+.diamond:before,
+.diamond:after {
+  content: '';
+  position: absolute;
+  width: 0;
+  border: 50px solid transparent;
+}
+
+.diamond:before {
+  border-bottom-color: #078ecd; /* White color for top triangle */
+  top: 25px;
+}
+
+.diamond:after {
+  border-top-color: rgb(255, 0, 76); /* Pink color for bottom triangle */
+  bottom: 25px;
+}
+
+/* Sparkling animation */
+.sparkle1 {
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  background-color: #cd4607;
+  animation: sparkleAnimation 2s infinite;
+}
+
+.sparkle2 {
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  background-color: #c3cd07;
+  animation: sparkleAnimation 2s infinite;
+}
+
+.sparkle3 {
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  background-color: #07cd28;
+  animation: sparkleAnimation 2s infinite;
+}
+
+.sparkle4 {
+  position: absolute;
```

Comment:
CSS properties: `position`, `width` , `height`, and animation is repetitive for sparke1, sparkle2, and sparkle3. You can create a common class:

`.sparkle{. 
  position: absolute;
  width: 10px;
  height: 10px; 
 animation: sparkleAnimation 2s infinite;
} `

`.sparkle1 { background-color: #cd4607; }`

and use the `sparkle sparkle1` classes in HTML